style.css

```diff
@@ -401,10 +473,12 @@ footer h3 {
 }
 
 #about_me {
-  height: 130vh;
-  margin: 0 0 0 16px;
+  height: 180vh;
+  margin: 3px;
   display: flex;
   flex-direction: column;
+  background-color: #fff;
+  border-radius: 0 10% 0 0;
 }
```

Comment:
As requested by the previous reviewer, please make sure to apply the desktop design displayed on Figma 😉. Please note that, currently, you still have your mobile version styling:

| Figma design  | Your design |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/95243996/201199537-43017528-be50-4938-968a-00cd415e403f.png" />  | <img src="https://user-images.githubusercontent.com/95243996/201199418-a5acc49b-b57e-4043-b3c1-e2b205381835.png" />  |